### ISSUE TITLE
Notification if a list was shared

### ISSUE THREAD

**Issue by jakobroehrl** [nextcloud/tasks#614]
It would be great, if the user gets a notification if a task list was shared to him.

**Comment by raimund-schluessler** [comment]
I think this would be a nice enhancement, but this should go into server imho. Especially since this is probably also concerning sharing of calendars. @jancborchardt @georgehrke for their opinion.

**Comment by jancborchardt** [comment]
Yes, likely @georgehrke knows what to do here. And yes, there should be a sharing notification and likely also previously a way to accept/reject the share in the first place.

@jakobroehrl keep in mind not all of our users are men, so it’s better to simply use they/them instead of he/him when referring to them. Thanks! :)

**Comment by georgehrke** [comment]
Yes, this definitely belongs in the server.
